docs/zkapps/o1js/interact-with-mina.md

```diff
@@ -420,43 +486,15 @@ await pendingTx.wait();
 // our account updates are applied on chain!
 ```
 
-Apart from `Mina.Metwork`, you can also use a mocked "Mina instance" for local testing:
+In addition to `Mina.Metwork`, you can also use a mocked "Mina instance" for local testing:
 
 ```ts
 const Local = Mina.LocalBlockchain();
 Mina.setActiveInstance(Local);
 ```
 
-Doing this means setting up a fresh, local ledger, which is pre-filled with a couple of accounts with funds on them that you have access to. "Sending" a transaction here just means applying your account updates to that local Mina instance. This is helpful for testing, especially because account updates go through the same validation logic locally that they would on-chain.[^2]
-
-[^2]: Fun fact: `LocalBlockchain` literally uses the same OCaml code for transaction validation and application that the Mina node uses; it's compiled to JS with [js_of_ocaml](https://github.com/ocsigen/js_of_ocaml).
-
-<!-- TODO commenting this out as it overlaps with the testing section
-
-Everything else works the same locally as it would when interaction with a real network. You still need to properly sign your transactions and attach proofs, and then send them with `tx.send()` to apply them locally.
-
-To help you iterate quicker when testing, there's an option to disable proof creation: `Mina.LocalBlockchain({ proofsEnabled: false })`. You can use it like this:
-
-```ts
-// local mock Mina instance
-const proofsEnabled = false;
-const Local = Mina.LocalBlockchain({ proofsEnabled });
-Mina.setActiveInstance(Local);
-
-// only need to compile smart contract when proofs are enabled
-if (proofsEnabled) await MyContract.compile();
-
-const tx = await Mina.transaction(sender, () => {
-  // ...
-});
-await tx.prove();
-tx.sign([senderKey]);
-
-await tx.send();
-
-// our accounts updates are applied locally!
-```
+Doing this means setting up a fresh, local ledger that is pre-filled with a couple of accounts with funds that you have access to. "Sending" a transaction here just means applying your account updates to that local Mina instance. This is helpful for testing, especially because account updates go through the same validation logic locally that they would on-chain.
 
-Note that with `proofsEnabled: false`, you still need to call `await tx.prove()`. The only change is that `.prove()` creates a dummy proof instead of a real one, and runs in 0 seconds. Also, `tx.send()` will skip the usual step of verifying your proof. -->
+Fun fact: The `LocalBlockchain` instance literally uses the same OCaml code for transaction validation and application that the Mina node uses; it's compiled to JS with [js_of_ocaml](https://github.com/ocsigen/js_of_ocaml).
```

Comment:
Don't we use footnotes? 😅 Like the one above, this feels more like footnote content to me (but having it in the main text works as well, in this case)

Reply:
no footnotes, please :)